src/annotator/test/guest-test.js

```diff
@@ -609,29 +627,55 @@ describe('Guest', () => {
   });
 
   describe('#createAnnotation', () => {
-    it('adds metadata to the annotation object', () => {
+    it('adds document metadata to the annotation', async () => {
       const guest = createGuest();
       const annotation = {};
 
-      guest.createAnnotation(annotation);
+      await guest.createAnnotation(annotation);
 
-      return timeoutPromise().then(() => {
-        assert.equal(annotation.uri, fakeDocumentMeta.uri());
-        assert.deepEqual(annotation.document, fakeDocumentMeta.metadata);
-      });
+      assert.equal(annotation.uri, fakeDocumentMeta.uri());
```

Comment:
This test, while valid, is testing the mutated param, but not an actual returned variable. I think the function should either return nothing so it's very clear its mutating the param, or it should test the thing it actually is returning.

`const annotation = await guest.createAnnotation({});`